ophyd/signal.py

```diff
@@ -158,6 +156,7 @@ def _repr_info(self):
 
     def get(self, **kwargs):
         '''The readback value'''
+        logger.info('%s get() returning ', self, self._readback)
```

Comment:
On a phone call, @jklynch and I realized that OphydObj is creating a separate logger for every instance of OphydObj, a pattern we have since learned to avoid, in favor of using `LoggerAdapter` to wrap _one_ `Logger` with some additional context (in this case, `self.name`). This can kind of thing could be changed to

```py
self.log('get() returning %s', self._readback)
```

where `self.log` is a `LoggerAdapter` that wraps up the relevant context (`self.name`, maybe other stuff too...). Downstream, a `Formatter` can incorporate that extra context in the final log message.

Reply:
...To be clear, all of the above was just to illustrate how to thread `self.name` into log messages. In this _particular_ case (logging on every `get()`) I think we should probably strip this out and rely on messages right next to where the I/O occurs... `self._read_pv.get` and similar.